### PR TITLE
add datetimes to associations

### PR DESCRIPTION
**Pull request by jason2e**
also added a chargedwith

### CODE REVIEW

File: demojustice.yaml [hunk 1]
```diff
@@ -592,7 +603,34 @@ associationDefinitions:
...
+      general.datetime:
+        type: "general.datetime"
+        column: "ArrestDate"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["MM/dd/yy", "M/dd/yy", "MM/d/yy", "M/d/yy", "MM/dd/yyyy", "M/dd/yyyy", "MM/d/yyyy", "M/d/yyyy"]
+          timezone: "America/Los_Angeles"
+    name: "chargedwith2"
 
   chargedwith2:
```
Comment (jokedurnez) [inline demojustice.yaml]: duplicate key

File: demojustice.yaml [hunk 2]
```diff
@@ -675,6 +713,13 @@ associationDefinitions:
...
           columns: ["SubjectIdentification", "ChargeID"]
           separator: "_"
+      general.datetime:
+        type: "general.datetime"
+        column: "ArrestDate"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["MM/dd/yy", "M/dd/yy", "MM/d/yy", "M/d/yy", "MM/dd/yyyy", "M/dd/yyyy", "MM/d/yyyy", "M/d/yyyy"]
+          timezone: "America/Los_Angeles"
     name: "suspectedof"
```
Comment (jokedurnez) [inline demojustice.yaml]: is this chargedwith or suspected of?

**Review by jokedurnez — CHANGES_REQUESTED** [review]
Just some confusion about chargedwith2.  Otherwise looks good !

File: demojustice.yaml (hunk 1 above)
Comment (jason2e) [reply demojustice.yaml]: odd, why didn't that throw an error in shuttle...

**Pull request merged**